src/server/enrichment-map/emap/generateCys.js

```
@@ -20,7 +20,9 @@ const _ = require('lodash');
 
 // input ["GO:1902275", "GO:2001252", "GO:1905269", "GO:0051053"]
 // returns a cytoscape object
-const generateCys = (pathwayIdList) => {
+const generateCys = (pathwayIdList, cutoff) => {
+  if (cutoff === undefined) { cutoff = 0.375; }
+  if (cutoff < 0 || cutoff > 1) { return 'ERROR: cutoff out of range [0,1]';}
```

You should throw an error instead of returning a string